README.md

```diff
@@ -302,6 +303,28 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+#### grcov with GitLab
+
+Here is an example of a `.gitlab-ci.yml` file for source-based coverage, using
+GitLab's [test coverage visualization feature](https://docs.gitlab.com/ee/user/project/merge_requests/test_coverage_visualization.html):
+
+```yaml
+test:
+  image: rust:latest
+  variables:
+    RUSTC_BOOTSTRAP: 1
+    RUSTFLAGS: -Zinstrument-coverage
+  script:
+    - rustup component add llvm-tools-preview
+    - curl -sSL https://github.com/mozilla/grcov/releases/latest/download/grcov-v0.8.6-x86_64-unknown-linux-gnu.tar.gz | tar -xz
+    - cargo build --verbose
```

Comment:
And why `--verbose`? To have better logs in case anything fails? I think this isn't really helpful, but that's personal preference.

Reply:
Using `--verbose` is consistent with everything else in the README as well.